### PR TITLE
fix: correct the hyperlinks related to JSON-RPC.

### PR DESCRIPTION
**Pull request by aganisgash**
It's kind of annoying to have the wrong direction.

sorry for the json file, the diff mode looks weird but I only modify the related content, if these json files are automaticly generated, you can tag me for the recovery or just directly delete the json file. : ]

### CODE REVIEW

File: testdata/etherscan/0x71356E37e0368Bd10bFDbF41dC052fE5FA24cD05/metadata.json
Comment (mattsse) [inline testdata/etherscan/0x71356E37e0368Bd10bFDbF41dC052fE5FA24cD05/metadata.json:1]: why was this changed?

Comment (aganisgash) [reply testdata/etherscan/0x71356E37e0368Bd10bFDbF41dC052fE5FA24cD05/metadata.json:1]: I updated all the outdated hyperlinks which is expired. For these three JSON files, I only changed the relevant links content. You can use a file comparison tool.  As described in my file notes, if these three files are auto-generated based on other files, then there's no need to modify them here. What do you think?

**Review by DaniPopes — APPROVED** [review]
Thanks! I reverted the changes in testdata since they are not user-facing in any way.

File: testdata/etherscan/0x71356E37e0368Bd10bFDbF41dC052fE5FA24cD05/metadata.json
Comment (aganisgash) [reply testdata/etherscan/0x71356E37e0368Bd10bFDbF41dC052fE5FA24cD05/metadata.json:1]: oh sorry, I just noticed that the test code is tied with contract address,  I will delete it thanks

**Comment by aganisgash** [comment]
thanks @DaniPopes just finished the delete, haven't push yet, hanks for your kind help
